src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -824,9 +824,11 @@ substitutions:
   ".*Switch state":
     "off": 0
     "in": 1
+    "on": 1
```

Comment:
please also update the tests/unit/mappings/test_value_mapping.py and the docs/converters/tabular_converter.md

Reply:
@nitish the documentation is still left: docs/converters/tabular_converter.md

Reply:
i would argue so because people will just blindly copy that into their own projects. we don't guarantee the docs to be up to date but they should be